cluster-install/install.sh

```diff
@@ -1500,8 +1527,14 @@ step_check_config() {
     add_to_config_builder "CONNECTOR_AUTOSUPPORT_ENROLLED"
     add_to_config_builder "CONNECTOR_AUTOSUPPORT_URL"
 
+    # Add our default labels
+    local -r label_indent="    "
+    local -a default_labels=("app.kubernetes.io/created-by=astra-cluster-install-script")
+    _PROCESSED_LABELS="$(process_labels_to_yaml "${default_labels[*]}" "$label_indent")"
```

Comment:
This adds a label which will allow users (and our pipeline) to identify/select all the resources the script created. It's added alongside any custom user labels:
```
    labels:
      app.kubernetes.io/created-by: astra-cluster-install-script
      testLabel: hello
      testLabelTwo: helloTwo
```